types/lodash/lodash-tests.ts

```diff
@@ -6330,27 +6489,27 @@ namespace TestPartition {
 //     {
 //         let result: _.LoDashImplicitArrayWrapper<AbcObject>;
 //
-//         result = _(array).map<AbcObject>('d.0.b');
-//         result = _(array).map<AbcObject>(['d', 0, 'b']);
+//         result = _(array).map('d.0.b');
```

Comment:
I think you should actually remove these lines, and add proper `_.map` tests above (line ~6200). You can use these tests as inspiration before you delete them :)

Reply:
Yeah I would ignore that for now. If you want, you can review it later and create a separate PR (or at least create an issue so someone else can create a PR).

I'm not 100% convinced that `_.map` should accept an array of strings. If it does, then all `iteratees` should accept an array of strings. Again, this is a problem for a separate PR, so probably delete those tests for now.

Reply:
I just now reverted my change to support PropertyPath in `_.map`.